building/gits/optee_os.rst

```diff
@@ -449,8 +450,41 @@ disable feature ``CFG_CRYPTO_GCM``:
     $ export CFG_CRYPTO_GCM=n
     $ make
 
-Configuration variables may then be used directly in ``sub.mk`` to trigger
-conditional compilation:
+When a configuration variable is introduced, the proper way to enable it by
+default is to write ``CFG_XXX ?= y`` in some ``.mk`` file. If it should be
+default disabled instead, simply do not set it to any value.
+
+If a platform needs to set a default value that is different from the one
+that is set in a global configuration file (typically, ``mk/config.mk``),
+it can do so in ``core/arch/$(arch)/plat-$(platform)/conf.mk``. For
+example:
+
+.. code-block:: text
+
+   # CFG_FOO is enabled by default except for PLATFORM=xyz
+   # Override is allowed
+
+   # In core/arch/$(arch)/plat-xyz/conf.mk
+   CFG_FOO ?= n
+
+   # In mk/config.mk
+   CFG_FOO ?= y
+
+.. code-block:: text
+
+   # CFG_FOO is enabled by default except for PLATFORM=xyz
```

Comment:
maybe add `... for PLATFORM=xyz that requires CFG_FOO disabled`

Reply:
OK